zer0-os/zOS
# feat: start a direct message conversation with existing zero users by searching their name

### What does this do?
When typing into the search conversations input, a new section has been added that will display users that match the string that is entered. 

### Why are we making this change?
The idea is that a user doesn't have to send an invite link to another user to search, find and start a direct conversation with.

### How do I test this?
Type characters into the search conversation input and click a result that appears. This should open the direct message window that enables you to send a message to that user.

https://github.com/zer0-os/zOS/assets/39112648/f052582a-599c-4192-b9ad-99b55c076adb


### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?
